openpype/hosts/fusion/plugins/publish/extract_render_local.py

```diff
@@ -33,6 +29,7 @@ def process(self, instance):
             )
         )
 
+        # Generate the frame list
```

Comment:
Couldn't the logic here also just re-use the Expected Frame list from the Collector?

Reply:
Yes, yes it could and should... Fixed!